src/WinGetUtil/WinGetUtil.h

```diff
@@ -22,6 +22,7 @@ extern "C"
         Default = 0,
         SchemaValidationOnly = 0x1,
         ErrorOnVerifiedPublisherFields = 0x2,
+        InstallerValidation = 0x4,

```

Comment:
>InstallerValidation = 0x4, [](http://example.com/codeflow?start=8&length=26)

nit: if you want to keep this option under WinGetValidateManifestOption, can you name it InstallerValidations to avoid name confliction? Or if we're to remove this after V3 manifest validation integration, you can ignore this comment

Reply:
Updated to `InstallerValidations` to prevent conflicts 👍